sktm/__init__.py

```diff
@@ -92,9 +92,7 @@ def add_pw(self, baseurl, pname, lpatch=None, restapi=False, apikey=None,
         Args:
             baseurl:        Patchwork base URL.
             pname:          Patchwork project name.
-            lpatch:         Last processed patch. Patch ID, if adding an XML
-                            RPC-based interface. Patch timestamp, if adding a
-                            REST-based interface. Can be omitted to
+            lpatch:         Last processed patch. Patch ID. Can be omitted to
```

Comment:
Do you think saying `ID of the last processed patch.` here would read better?

Reply:
Absolutely!